terraform/datadog/elb.tf

```diff
@@ -3,11 +3,11 @@ resource "datadog_monitor" "abnormal_api_latency_cc" {
   type    = "query alert"
   message = "${format("{{#is_alert}}We're experiencing >= {{threshold}} change in ELB Latency.{{/is_alert}} \n{{#is_warning}}We're experiencing >= {{warn_threshold}} change in ELB Latency.{{/is_warning}} \n\nVisit the [Team Manual > Responding to alerts > API Latency](%s#api-latency) for more info.", var.datadog_documentation_url)}"
 
-  query = "${format("avg(last_15m):anomalies(avg:aws.elb.latency{name:%s-cf-cc}, 'basic', 2, direction='both') > 0.3", var.env)}"
+  query = "${format("avg(last_1d):avg:aws.elb.latency{name:%s-cf-cc} > 1", var.env)}"
```

Comment:
I appreciate setting this aggregation over 1 day smooths out spikes but do we feel this is going to trigger quick enough? For example, if our regular latency is 0.1 seconds it'd take an hour of requests that averaged ~21.7 seconds before this triggered.

Reply:
(at the critical level)

Reply:
I'm _not sure_ if that's right. DataDog works in mysterious ways...

If I attempt to set something below 1h window, I get the warning:

> The selected metric provides data at low frequencies, we strongly recommend alerting on a 1 hour timeframe

It's a difficult one to get right really... Following the guide from DD and taking our concerns into the factor, with the current thresholds the following setup:

> Trigger when the metric is `above` the threshold `at least once` during the last `1 hour`

would trigger 27 warnings and 3 alerts today...  This is too noisy in my opinion and I don't think it's reasonable to increase the thresholds above 1 second.

Your point is still valid, therefore I'd prefer to delay/block merging this PR and have a general tech talk with the team. Maybe someone could clarify things for us or we come up with a different solution.